ui/src/main/ui/src/dashboard/dashboard-service.js

```diff
@@ -2,11 +2,12 @@ angular.module('hwk.dashboardModule').service('hwk.dashboardService', ['$resourc
   function ($resource) {
     'use strict';
 
+    var host = '';
```

Comment:
This also would need to be changed on triggers-service.js and actions-service.js as well.
As this is for development mode, I was thinking in some global parameter passed via 'grunt server' and automatically detected.
I haven't played yet with it.
Feel free to propose something in that direction.